nemo-physical/src/function/definitions/string.rs

```diff
@@ -11,6 +12,57 @@ use super::{
     BinaryFunction, FunctionTypePropagation, NaryFunction, TernaryFunction, UnaryFunction,
 };
 
+/// Unicode friendly version of [String] find
+///
+/// Returns the index of the first occurrence of the second parameter in the first parameter.
+/// or `None` if the second parameter is not found.
+fn unicode_find(parameter_first: &String, parameter_second: &String) -> Option<usize> {
```

Comment:
These should really not be called `parameter_first` and `parameter_second`, but something descriptive, e.g., `needle` and `haystack` (or maybe `pattern` and `string`; in any case the name should already make clear which one is which – the order is already apparent from the signature anyways.)

Reply:
I completely agree. I named the parameters that to be more consistent with the other method headers. Can it break the convention because it takes in `String`s instead of `AnyDataValue`s? Perhaps because it is essentially a utility method?